CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.3.0
```

Comment:
Sort of late, but this should be 3.0.0.

Reply:
Fix here... the pubspec wasn't updated either: https://github.com/dart-lang/mockito/pull/89